charm/ksql/metadata.yaml

```diff
@@ -17,6 +17,14 @@ requires:
 provides:
   website:
     interface: http
+storage:
+  state:
+    type: filesystem
+    description: Directory where KSQL state is stored
+    minimum-size: 20G
```

Comment:
This is a little on the high side, if one just wanted to test storage. Minimum size for kafka and zookeeper are 20M. This isn't necessarily a recommended minimum; it's more of an absolute minimum of what is allowed to attach in order for the service to work at all. This is typically small; postgresql charm minimum-size is 5M. I think 20M would be OK here.